### PR TITLE
Move lsp4ij to its own repository

### LINKED ISSUE #556: Use separate LSP4IJ component
Use the language server support in the component LSP4IJ in https://github.com/MicroShed/lsp4ij and delete the similar files in this plugin. 

Part of #420

### PR DESCRIPTION
**Pull request by turkeylurkey**
In build.gradle some dependencies now say "exclude group: 'org.eclipse.lsp4j'". This was needed when using lsp4ij as a plugin and now it is just good hygiene. I added lsp4j as a dependency once again just for clarity even though lsp4ij also includes it.

Fixes #556

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -361,6 +361,12 @@
...
     </actions>
     <!-- Point to LSP4IJ xml -->
+    <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="/META-INF/lsp4ij.xml"
+                xpointer="xpointer(/idea-plugin/*)"/>
+    <!-- Point to LSP xml -->
     <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="/META-INF/lsp.xml"
                 xpointer="xpointer(/idea-plugin/*)"/>
-</idea-plugin>
\ No newline at end of file
+<!--    <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"-->
```
Comment (turkeylurkey) [inline src/main/resources/META-INF/plugin.xml:369]: Take out attempt.

Comment (turkeylurkey) [reply src/main/resources/META-INF/plugin.xml]: fixed

File: DEVELOPING.md [hunk 2]
```diff
@@ -16,7 +16,10 @@
 
 This extension is built using the [gradle-intellij-plugin](https://github.com/JetBrains/gradle-intellij-plugin/).
 
-1. Clone this repository: `git clone git@github.com:OpenLiberty/liberty-tools-intellij.git && cd liberty-tools-intellij`
+1. Clone this repository: `git clone git@github.com:OpenLiberty/liberty-tools-intellij.git`
+2. Clone the lsp4ij repository: `git clone git@github.com:MicroShed/lsp4ij.git`
+3. Build lsp4ij: `./gradlew jar`
+4. Save it in your local Maven repository: `mvn install:install-file -Dfile=build/libs/lsp4ij-0.0.5.jar -DgroupId=org.microshed -DartifactId=lsp4ij -Dversion=0.0.5 -Dpackaging=jar -DgeneratePom=true;`
```
Comment (mrglavas) [inline DEVELOPING.md:22]: Should update these instructions to just `./gradlew publishToMavenLocal` as we're doing in the GitHub action.

**Review by mrglavas — APPROVED** [review]
Looks good. Thanks.

File: DEVELOPING.md (hunk 2 above)
Comment (turkeylurkey) [reply DEVELOPING.md]: Ok

**Review by TrevCraw — APPROVED** [review]

**Pull request merged**